server/middleware/user.js

```diff
@@ -6,10 +6,10 @@ const { FormSubmissionError } = require('../models/error');
 async function addToTeam(req, _, next) {
```

Comment:
```suggestion
async function addUsersToTeam(req, _, next) {
```

Would this not be more descriptive?

Reply:
I have changed this